nixos/tests/postgresql-wal-receiver.nix

```diff
@@ -32,6 +32,10 @@ let
       meta.maintainers = with lib.maintainers; [ pacien ];
 
       nodes.machine = { ... }: {
+        systemd.tmpfiles.rules = [
+          "d /var/cache/wals 0750 postgres postgres - -"
+        ];
```

Comment:
might be worth using `tmpfiles.settings` instead to prevent more PR's like this https://github.com/NixOS/nixpkgs/pull/343796